### PR TITLE
Bad branch: Add images and informations about chicken breeds

### PR DESCRIPTION
**Pull request by barboradankova**
Add images and informations about chicken breeds.

### CODE REVIEW

File: chicken_breed_names.txt [hunk 1]
```diff
@@ -1,6 +1,5 @@
...
-	1. Sumatra Chicken
-	2. Yokohama Chicken
-	3. Sultan Chicken
-	4. Chicken Nuggets
-//TODO Add brahma chicken
+My favourite chicken breeds:
+1. Sumatra Chicken
+2. Yokohama Chicken
+3. Sultan Chicken
+4. Chicken Nuggets
```
Comment (heyshowshana) [inline chicken_breed_names.txt:5]: Thanks for fixing this file, I would also like to remove `4. Chicken Nuggets`

**Review by heyshowshana — COMMENTED** [review]
Great, thanks!
Please lets also remove the `chicken_nuggets` image.

File: chicken_breed_names.txt [hunk 2]
```diff
@@ -3,3 +3,4 @@ My favourite chicken breeds:
 2. Yokohama Chicken
 3. Sultan Chicken
 4. Chicken Nuggets
+5. Brahma Chicken
```
Comment (heyshowshana) [inline chicken_breed_names.txt:6]: Amazing, thanks!

**Comment by heyshowshana** [comment]
A summary of what we should do to be able to merge the pull request:

- [ ] Remove Pigeon and Chicken nuggets from both, the list and images
- [ ] Have Brahma chicken in the list

**Comment by barboradankova** [comment]
Now, conflicts are resolved according to your requirements.

**Pull request merged**